Core/Security/VOMS.py

```diff
@@ -173,8 +175,8 @@ def getVOMSProxyInfo(self, proxy, option=False):
           lines.append("attribute : %s" % data['attribute'])
         now = Time.dateTime()
         left = (data['notAfter'] - now).total_seconds()
-        h = int(left / 3600)
-        m = int(left / 60) - h * 60
+        h = int(left // 3600)
```

Comment:
insn't the output of `//` already an `int` ?

Reply:
Yes, here `int` could be removed. It was there before, so I didn't touch it, but yes it could (somewhere else I removed `float`)